src/styles/_homepage.scss

```diff
@@ -72,3 +72,12 @@
 .homepage-feature {
   margin-top: $spacing-09;
 }
+
+// OVERRIDES
+.resource-card-group .bx--resource-card--dark {
+  border-color: #000;
+}
+
+.resource-card-group .bx--col-lg-4:nth-child(even) .bx--resource-card--dark {
+  border-left-color: #000;
+}
```

Comment:
This is a temporary overrride. These styles are coming from the GTC, so they need to be fixed there.